AutoPBW.WPF/MainWindow.xaml.cs

```diff
@@ -731,6 +731,11 @@ public void OnTurnFileChanged_MainThread(object source, FileSystemEventArgs e)
 							game.UploadTurn();
 							anyUploaded = true;
 						}
+						catch (IOException ioex) when ((ioex.HResult & 0x0000FFFF) == 0x0020)

```

Comment:
Wait, you can put conditions on catch clauses? That's cool!

Reply:
Yeah I just found out about it too! Very useful here because there's no "Sharing Violation Exception" to catch!